public/app/features/dashboard-scene/edit-pane/DashboardEditPane.tsx

```diff
@@ -104,6 +106,12 @@ export function DashboardEditPaneRenderer({ editPane, isCollapsed, onToggleColla
     };
   }, [editPane]);
 
+  useEffect(() => {
+    if (isCollapsed && editPane.state.selectedObject?.resolve()) {
+      editPane.clearSelection();
+    }
+  }, [editPane, isCollapsed]);
```

Comment:
The purpose of this here is to avoid an awkward kind of behavior where if there is a selection and the user closes the edit pane split, since that will be closed and a selection exists, it will immediately open the overlay. 

This is a quick fix for this situation, shown here:

https://github.com/user-attachments/assets/1ec077da-e3ab-4eca-b63f-41c35fec7823

But I'm wondering if it's the way to go? I'm thinking in terms of multiselect and if it will eventually also enable multi dragNdrop? in which case the selection should not be cleared and the overlay should maybe also not pop up since we want screen space?

idk...

Reply:
Now I'm thinking about going back to the initial hover idea @bfmatei had and add an extra step to the overlay to show on hover